### ISSUE TITLE
Cannot use queue with the simulator/console

### ISSUE THREAD

**Issue by ShaiBer** [winglang/wing#2087]
### I tried this

Wrote this code while the console was open:

```ts
bring cloud;

let bucket = new cloud.Bucket();
let queue = new cloud.Queue();

queue.add_consumer(inflight (message: str) => {
  bucket.put("wing.txt", "Hello, ${message}");
});
```

### I expected this:

Get a visualization of the code

### Instead, this happened

Got this error:

```
Failed to start simulator.

Error: Type wingsdk.sim.EventMapping not implemented by the simulator.
```

### Is there a workaround?

No

### Component

Wing Console

### Wing Version

0.11.4

### Wing Console Version

0.2.32

### Node.js Version

v18.12.1

### Platform(s)

MacOS

### Anything else?

1. If I compile the code to the sim or to aws it works
2. From the error it seems like a problem with the simulator

### Community Notes

* Please vote by adding a 👍 reaction to the issue to help us prioritize.
* If you are interested to work on this issue, please leave a comment.

**Comment by eladb** [comment]
Fixed formatting

**Comment by eladb** [comment]
Suspect: https://github.com/winglang/wing/pull/2054

**Comment by staycoolcall911** [comment]
Fixed by #2090
